src/components/Checkout/OrderTotals.vue

```diff
@@ -22,9 +26,81 @@
 				Apply
 			</button>
 		</div>
-		<div class="order-total columns small-12">
-			<strong>Total: <span class="total-value">{{ orderTotal }}</span></strong>
+
+		<div v-if="showPromoCreditTotal">
+			<div class="order-total">
+				<template v-if="availablePromoTotal">
+					{{ availablePromoTotal }}
+				</template>
+				<kv-button
+					class="text-link"
+					id="promo_name"
+					v-if="promoFundDisplayName"
+				>
+					{{ promoFundDisplayName }}
+				</kv-button> promotion: <span class="total-value">({{ appliedPromoTotal }})</span>
+				<button
+					v-if="showRemoveActivePromoCredit"
+					class="remove-credit"
+					@click="verifyRemovePromoCredit"
+				>
+					<kv-icon class="remove-credit-icon" name="small-x" :from-sprite="true" title="Remove Credit" />
+				</button>
+				<button
+					v-if="showApplyActivePromoCredit"
+					class="apply-credit small-text"
+					@click="applyPromoCredit"
+				>
+					Apply
+				</button>
+			</div>
+			<kv-tooltip
+				class="tooltip"
+				controller="promo_name"
+				v-if="promoFundDisplayDescription"
+			>
+				{{ promoFundDisplayDescription }}
+			</kv-tooltip>
+		</div>
+
+		<div class="order-total">
+			<strong>
+				<template v-if="!showPromoCreditTotal">Total: </template>
+				<template v-else>Total Due: </template>
+				<span class="total-value">{{ creditAmountNeeded }}</span>
+			</strong>
```

Comment:
This is our original and permanent Total line item with an adjust value variable name and copy clarifications in the promo campaign view.

Reply:
👏 
I feel that we've needed something like this (in combination with that Order Total line) in regular checkout for quite some time now, so I'm really glad to see it making an appearance here!